api/app/settings/common.py

```diff
@@ -467,3 +467,8 @@
 
 # DynamoDB table name for storing identities
 IDENTITIES_TABLE_NAME_DYNAMO = env.str("IDENTITIES_TABLE_NAME_DYNAMO", None)
+
+# Set this to enable create organisation for only superusers
+ONLY_SUPERUSERS_CAN_CREATE_ORGANISATIONS = env.bool(
```

Comment:
Being picky now but I think we should rename this setting to `RESTRICT_ORG_CREATE_TO_SUPERUSERS`

Reply:
`RESTRICT_ORGANISATION_CREATE_TO_SUPERUSERS` no?